util/forwardedhosthandler_test.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	mockHttp "content-ui-service/mocks/net/http"
```

Comment:
travis build is red, because import path is wrong ..
maybe just copy over the mock first and do a clean go:generate later on ..